from ga4gh.drs.exceptions.drs_exception import DRSException

class CLIException(DRSException):
    pass